src/processor/dataSample.ts

```diff
@@ -94,7 +94,7 @@ export default function dataSample(seriesType: string): StageHandler {
                 const size = Math.abs(extent[1] - extent[0]) * (dpr || 1);
                 const rate = Math.round(count / size);
 
-                if (rate > 1) {
```

Comment:
Using `isFinte(rate)` may be better

Reply:
> Using `isFinte(rate)` may be better

Sure, is it better for `if (isFinite(rate) && rate > 1)` ?

Reply:
Yes I think so